Changelog.md

```diff
@@ -26,7 +32,9 @@ Other Changes
 [#437]: https://github.com/NLnetLabs/routinator/pull/437
 [#443]: https://github.com/NLnetLabs/routinator/pull/443
 [#444]: https://github.com/NLnetLabs/routinator/pull/444
+[#456]: https://github.com/NLnetLabs/routinator/pull/456
```

Comment:
This line is unrelated to this PR as far as I can see, it was just missed from PR #456. If the intention is to strictly relate changes to the issues they address then perhaps this and the reference above to 456 should be pulled out of this PR and pushed separately?

Reply:
I don’t feel like we need to be too concerned about the changelog. As long as it has everything noteworthy upon release.